routes/auth.php

```diff
@@ -15,14 +15,14 @@
                 ->name('register');
 
 Route::post('/register', [RegisteredUserController::class, 'store'])
-                ->middleware('guest');
+                ->middleware('guest')->name('register.post');
 
 Route::get('/login', [AuthenticatedSessionController::class, 'create'])
                 ->middleware('guest')
                 ->name('login');
 
 Route::post('/login', [AuthenticatedSessionController::class, 'store'])
-                ->middleware('guest');
+                ->middleware('guest')->name('login_post');
```

Comment:
üstteki gibi yapalım tutarlı olsun
```suggestion
                ->middleware('guest')->name('login.post');
```